.travis.yml

```diff
@@ -20,6 +20,13 @@ cache:
 addons:
   firefox: latest
 
+before_install:
+  - sudo apt-get update -qq
```

Comment:
Will this be part of build instructions as well. If so, should we move these into install_thrift.sh script?

Reply:
updated readme, this should still be here.